components/eamxx/src/share/io/eamxx_io_utils.cpp

```diff
@@ -231,4 +231,53 @@ create_diagnostic (const std::string& diag_field_name,
   return diag;
 }
 
+std::pair<std::string, std::string>
+parse_field_alias (const std::string& field_spec)
+{
+  const std::string delimiter = ":=:";
```

Comment:
@mahf708 You could tell him to use utils in `ekat_string_utils.hpp` for this kind of string split/strip manipulation.